### DISCUSSION TITLE
Building with fortran

### DISCUSSION THREAD

**Discussion by mastino** [LLNL/Caliper#366] [Q&A]
Hello,

I am attempting to build Caliper 2.5 with fortran support and the fujitsu compiler. This may be a question for CMake folks, but it simply isn't building fortran support even though the cmake command reports Yes it found the compiler. Do you have any thoughts on this issue?

Thanks,
Brian

**Comment by daboehme** [comment]
Hi @mastino , do you see any error messages? What happens if you try to explicitly build some of the fortran components, e.g. `make fortran-example`? I don't have experience with or access to a Fujitsu system unfortunately.

**Reply by mastino** [reply]
The only error message is that caliper mod is missing

**Reply by daboehme** [reply]
That's weird. The Fortran module should be in `fortran/caliper_mod.mod` inside the build directory. Can you try `make VERBOSE=1 caliper-interface` and check for anything unusual in the output?

**Reply by mastino** [reply]
I found it in the build directory just now and caliper mod was there. I moved it over to the install directory and it seems to work so far

making the interface explicitly shows nothing out of the ordinary and make install afterwards doesn't install the fortran mod either

**Reply by daboehme** [reply]
Ok, strange, `make install` is supposed to move the fortran module into the install directory of course. This works for me.

**Reply by mastino** [reply]
Every other system I've used (probably over a dozen compiler OS combos) has not had this issue. It seems like the work around is fine, so I wouldn't worry about it too much
